packages/frontend/src/hooks/organisation/useOrganisationDeleteMultation.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiError } from '@lightdash/common';
+import { useMutation } from 'react-query';
+import { lightdashApi } from '../../api';
+import useToaster from '../toaster/useToaster';
+
+const deleteDashboard = async (id: string) =>
+    lightdashApi<undefined>({
+        url: `/org/${id}`,
+        method: 'DELETE',
+        body: undefined,
+    });
+
+export const useDeleteOrganisationMutation = () => {
+    const { showToastError } = useToaster();
+    return useMutation<undefined, ApiError, string>(deleteDashboard, {
+        onSuccess: async () => {
+            window.location.href = '/register';
```

Comment:
We can show a success toast just so they aren't confused.

Reply:
:thinking: show a toast for a few seconds and then redirect ? we can't leave the user in an invalid state (wrong org and logged out)

Reply:
ah, because the redirect forces a page reload ... right 🤔 forget this then 👍